target/linux/ipq806x/image/Makefile

```diff
@@ -92,6 +92,21 @@ define Device/ZyXELImage
 	IMAGE/sysupgrade.bin/squashfs := append-rootfs | pad-to $$$$(BLOCKSIZE) | sysupgrade-tar rootfs=$$$$@ | append-metadata
 endef
 
+define Device/askey_rt4230w-rev6
+	$(call Device/LegacyImage)
+	DEVICE_VENDOR := Askey
+	DEVICE_MODEL := RT4230W
+	DEVICE_VARIANT := REV6
+	SOC := qcom-ipq8065
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	DEVICE_PACKAGES := ath10k-firmware-qca9984-ct
+	KERNEL_IN_UBI := 1
+	IMAGES := nand-sysupgrade.bin
```

Comment:
Not `nand-sysupgrade.bin` --> `sysupgrade.bin`

Reply:
`Device/Default` already defines `sysupgrade.bin` so you can just drop the two lines. Or you can use `$(call Device/UbiFit)` as I mentioned in https://github.com/openwrt/openwrt/pull/3115#discussion_r447598459.

Reply:
Changing it from LegacyImage to UbiFit breaks compilation for some reason so I left it alone. I was able to remove the 2 lines without issues.